hack/make/.integration-daemon-stop

```diff
@@ -10,6 +10,8 @@ if [ ! "$(go env GOOS)" = 'windows' ]; then
 		if ! wait "$pid"; then
 			echo >&2 "warning: PID $pid from $pidFile had a nonzero exit code"
 		fi
+		root=$(dirname "$pidFile")/root
+		grep -qE "\b$root\b" /proc/mounts && umount "$root"
```

Comment:
It's better to not parse /proc/mounts before calling umount; easier to ignore non-zero exit code
```
umount "$root" || true
```

Reply:
Reason is, generating /proc/mounts is relatively expensive and sometimes error-prone so there is a chance you might not find your mount even if it's there.